tests/lenses/rust_wasm32_remove/src/lib.rs

```diff
@@ -0,0 +1,79 @@
+use std::collections::HashMap;
```

Comment:
question: why no copyright headers on rust files?

Reply:
Why would we want them here? I don't think we should discourage the use/copy-pasting of example lens modules

Reply:
Isn't this a test file?

Reply:
Yes.  And as such is a tested example of a working Lens module

Reply:
We have go test files that demonstrate examples that have headers.

Reply:
> We have go test files that demonstrate examples that have headers.

Yes, but those are also both of somewhat unknown value to me, and largely coupled to our test frameworks. This file here is an independent compilable, that is not even couple to Defra - it can be used by any Lens use-case.

Reply:
> Why would we want them here? I don't think we should discourage the use/copy-pasting of example lens modules.

Copyright don't prohibit the use/copy paste of code. The license headers in the files convey what responsibility both parties (author and consumer) have been each other, what liabilities, warranties, etc.

I'm more than happy to put all the rust example modules under a MIT license (like we do with the net code)

Reply:
@jsimnz In the short term do you want the normal Defra header on these? And if so why?

Reply:
As discussed we can assign the entire folder to a MIT license, and omit any file headers

Reply:
Has been done :) I had added a task to the PR description to avoid having to find this convo again